### PR TITLE
feat(connector): [Paybox] add connector template code

### PR DESCRIPTION
**Pull request by KiranKBR**
## Type of Change

- [x] New feature

## Description

Template code for Paybox

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [ ] This PR modifies application configuration/environment variables

## Motivation and Context

https://github.com/juspay/hyperswitch/issues/5489

## How did you test it?

This is template code so, no api testing required

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: scripts/add_connector.sh [hunk 1]
```diff
@@ -6,7 +6,7 @@ function find_prev_connector() {
...
-    connectors=(aci adyen adyenplatform airwallex applepay authorizedotnet bambora bamboraapac bankofamerica billwerk bitpay bluesnap boku braintree cashtocode checkout coinbase cryptopay cybersource datatrans dlocal dummyconnector ebanx fiserv forte globalpay globepay gocardless gpayments helcim iatapay itaubank klarna mifinity mollie multisafepay netcetera nexinets noon nuvei opayo opennode payeezy payme payone paypal payu placetopay plaid powertranz prophetpay rapyd razorpay shift4 square stax stripe threedsecureio trustpay tsys volt wellsfargo wise worldline worldpay zsl "$1")
+    connectors=(aci adyen adyenplatform airwallex applepay authorizedotnet bambora bamboraapac bankofamerica billwerk bitpay bluesnap boku braintree cashtocode checkout coinbase cryptopay cybersource datatrans dlocal dummyconnector ebanx fiserv forte globalpay globepay gocardless gpayments helcim iatapay itaubank klarna mifinity mollie multisafepay netcetera nexinets noon nuvei opayo opennode paybox paybox payeezy payme payone paypal payu placetopay plaid powertranz prophetpay rapyd razorpay shift4 square stax stripe threedsecureio trustpay tsys volt wellsfargo wise worldline worldpay zsl "$1")
```
Comment (deepanshu-iiitu) [inline scripts/add_connector.sh:9]: Why is paybox added twice here?

**Review by deepanshu-iiitu — CHANGES_REQUESTED** [review]

**Review by deepanshu-iiitu — DISMISSED** [review]

File: config/deployments/production.toml [hunk 2]
```diff
@@ -70,6 +70,7 @@ noon.key_mode = "Live"
 nuvei.base_url = "https://ppp-test.nuvei.com/"
 opayo.base_url = "https://pi-live.sagepay.com/"
 opennode.base_url = "https://api.opennode.com"
+paybox.base_url = "https://preprod-ppps.paybox.com/PPPS.php"
```
Comment (sai-harsha-vardhan) [inline config/deployments/production.toml:73]: Can we please use the production url here?

Comment (KiranKBR) [reply config/deployments/production.toml:73]: Yeah bro, merchant yet to provide production url

File: config/deployments/sandbox.toml
Comment (SanchithHegde) [inline config/deployments/sandbox.toml:1]: Please make the same changes in `config/deployments/integration_test.toml` as well.

File: config/deployments/production.toml (hunk 2 above)
Comment (SanchithHegde) [reply config/deployments/production.toml:73]: If this is going to be merchant specific, shouldn't we be ideally using a templated URL like we've done for Adyen, for instance?

https://github.com/juspay/hyperswitch/blob/540ef071cb238a56d52d06687226aab7fd0dfe68/config/deployments/production.toml#L26

**Review by SanchithHegde — APPROVED** [review]

**Review by SamraatBansal — APPROVED** [review]

**Review by sai-harsha-vardhan — APPROVED** [review]

**Pull request merged**